collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -327,7 +328,15 @@ public void onItemClicked(SortDialogAdapter.ViewHolder holder, int position) {
         }
     }
 
-    protected void showSnackbar(String result) {
+    protected void showSnackbar(@NonNull String result) {
         SnackbarUtils.showSnackbar(llParent, result);
     }
+
+    protected void setToolbarProgress(boolean visibility) {
```

Comment:
I think there might be a more descriptive name for this method. The purpose is to show or hide the ProgressBar. How about `showProgressBar(boolean show)` or `setProgressBarVisibility(boolean visibility)`? 

I haven’t looked closely enough to see how “Toolbar” fits into the naming.

Reply:
The thought behind adding 'Toolbar' to the method name was to differentiate between the normal progress dialog and this progress bar attached with the toolbar layout.

What do you think?